crypto/verifier.go

```diff
@@ -37,17 +35,6 @@ var (
 	}
 )
 
-// VerifyObject verifies the output of Signer.SignObject.
-func VerifyObject(pub crypto.PublicKey, obj interface{}, sig *sigpb.DigitallySigned) error {
```

Comment:
Suggestion: I'd probably leave this function here too, removing the newly-created object.go.